### ISSUE TITLE
I can't find the "Device Registration" section

### ISSUE THREAD

**Issue by epall** [actions-on-google/actions-on-google-testing-nodejs#4]
Working through the readme, I've created my test project but I can't find a "Device Registration" section in my Actions on Google console. The [SDK docs](https://developers.google.com/assistant/sdk/guides/library/python/embed/register-devicehttps://developers.google.com/assistant/sdk/guides/library/python/embed/register-device) talk about it, but the only sections under "Advanced Options" in my console are "Account linking", "Brand verification", and "Backend services". I tried enabling billing for the project, but that doesn't seem to have had an effect.

Where do I find this Device Registration UI?

**Comment by jneveux** [comment]
I believe you need to activate Google API service in GCP (Google Cloud Plateforme) on your project.

**Comment by epall** [comment]
Thanks, @jneveux!

I enabled the Google Assistant API in GCP, per the instructions.

Is there some other API that you're referring to? On a guess I tried enabling the "Actions API" but that seems to have had no immediate effect.

**Comment by Fleker** [comment]
When you create a project, you will see in the onboarding page the "Device Registration" template at the bottom of the page.

![image](https://user-images.githubusercontent.com/3291635/41870146-105c7c8a-7870-11e8-9d9f-0b02553981a2.png)


You may also be able to select navigate to the `deviceregistration` path in the URL after the project is created.

**Comment by epall** [comment]
Fantastic. Thanks!

**Comment by tkirshboim** [comment]
I just encountered this same issue myself when following the [How to get started](https://github.com/actions-on-google/actions-on-google-testing-nodejs/blob/d6283091d927289e46b07c5fb5ab8b9ec9768494/README.md#how-to-get-started) section in the README.

Navigating to `deviceregistration` after the project was created did not work for me. The only way I could to get to **Device Registration** was from the onboarding page (as suggested by @Fleker above). Once I pressed 'Skip' in the onboarding page I could not go back to it to change my choice or choose 'Device Registration' anywhere else. I had to recreate the project to get to the onboarding screen.

My suggestion:
I think that step 2 in the 'How to get started' section should mention that "Device Registration" should be selected in the onboarding screen.
Until that's done maybe this issue should be reopened so it's easily visible for others.

(and that's for building this project! I'm looking forward to trying it out.)

**Comment by CoreyCole** [comment]
@Fleker is there any way to get to the device registration page for an existing project?

**Comment by Fleker** [comment]
@CoreyCole it _might_ be at the bottom of the sidebar. It might not be. If not, that's fine. You just need a project that is linked to your Google account. This could be two separate projects.

**Comment by raficsm** [comment]
Hi, I am new to try smart home using firebase. I tried what is said in https://codelabs.developers.google.com/codelabs/smarthome-washer/#0

After finishing the above steps, when I tried adding my test device **[test]<project-name>** it simply redirects to the authorization url
https://us-central1-<project-id>.cloudfunctions.net/fakeauth
as mentioned in the steps and return to the device list page again without any errors. 

On verifying console log in firebase account, requestSync is not called yet and my device is not added in Home Graph (SYNC not called). Also, I enabled Home Graph API and didn't get any successful response.


But I don't what is the **agentUserId** - Can we give any string?
Client ID and Client Secret will be ABC123 and DEF456?

Authorization URL | https://us-central1-<project-id>.cloudfunctions.net/fakeauth
Token URL | https://us-central1-<project-id>.cloudfunctions.net/faketoken

I gave the above details as it is with replacing only the project id. I didn't anything in the code.
